packages/common-utils/src/metadata.ts

```diff
@@ -313,12 +313,12 @@ export class Metadata {
           query_params: sql.params,
           connectionId,
           clickhouse_settings: {
-            max_rows_to_read: String(
-              this.getClickHouseSettings().max_rows_to_read ??
-                DEFAULT_METADATA_MAX_ROWS_TO_READ,
-            ),
-            read_overflow_mode: 'break',
             ...this.getClickHouseSettings(),
+            // Max 15 seconds to get keys
+            timeout_overflow_mode: 'break',
+            max_execution_time: 15,
```

Comment:
@MikeShi42 you said "shorter query timeout" in  the ticket-  I guessed 15 seconds. Is this reasonable?